common/src/IO/NodeWriter.cpp

```diff
@@ -108,9 +108,9 @@ namespace TrenchBroom {
         m_world(world),
         m_serializer(MapStreamSerializer::create(m_world.format(), stream)) {}
 
-        NodeWriter::NodeWriter(const Model::WorldNode& world, NodeSerializer* serializer) :
+        NodeWriter::NodeWriter(const Model::WorldNode& world, std::unique_ptr<NodeSerializer> serializer) :
```

Comment:
Whoops! Nice catch!

Reply:
I was checking whether `NodeWriter` needed a defaulted destructor definition due to the unique_ptr, and it doesn't because `NodeWriter.h` include `NodeSerializer`. But there's no need for that include in the header file, so I'd ask that you move the include to `NodeWriter.cpp`, add a forward declaration and a defaulted destructor to `NodeWriter` to take care of deleting the serializer via the unique_ptr.

Reply:
Thanks!